components/modules/MentorCard/components/index.tsx

```diff
@@ -0,0 +1,235 @@
+import "twin.macro";
+import Image from "next/image";
+import { ReactElement, useMemo, useCallback, useState } from "react";
+import {
+    Stack,
+    StackDivider,
+    Badge as ChakraBadge,
+    AspectRatio,
+    IconButton,
+    IconButtonProps,
+} from "@chakra-ui/react";
+import { motion } from "framer-motion";
+import { Star, UserRectangle, ChatTeardropText, User, Heart } from "phosphor-react";
+import ColorThief from "color-thief";
+import { css } from "@emotion/css";
+
+/* -------------------------------------------------------------------------- */
+/*                                Profile Image                               */
+/* -------------------------------------------------------------------------- */
+
+// const rgbToHex = (r: number, g: number, b: number) =>
+//     `#${[r, g, b]
```

Comment:
Is there a reason you want to keep this commented-out `rgbToHex` function? If not we should probably just delete it.